pyscript.core/src/stdlib/pyscript/workers.py

```diff
@@ -0,0 +1,43 @@
+import js as _js
+from polyscript import workers as _workers
+
+_get = _js.Reflect.get
+
+
+def _set(script, name, value=""):
+    script.setAttribute(name, value)
+
+
+# this solves an inconsistency between Pyodide and MicroPython
+# @see https://github.com/pyscript/pyscript/issues/2106
+class _ReadOnlyProxy:
+    def __getitem__(self, name):
+        return _get(_workers, name)
+
+    def __getattr__(self, name):
+        return _get(_workers, name)
+
+
+workers = _ReadOnlyProxy()
+
+
+async def create_named_worker(src="", name="", config=None, type="py"):
```

Comment:
I thought @fpliger asked we hold on this `create_named_worker` function because YAGNI.

(But having said that, since we have it, there doesn't seem much point removing it... and perhaps it can hide in plain site and we just mention it as a convenience thing in our docs..?)

Reply:
well, I basically already wrote it and it helps in the docs explaining the difference between HTML named workers and `PyWorker` + it's the solution to anyone asking *why no name in PyWorker*, if ever.

The code looks simple enough to me so I'd rather keep in there than forget and search comments in issues ... does this make sense to you? If yes, I'll merge this, thanks!

Reply:
:+1: